utest/reporting/test_jsmodelbuilders.py

```diff
@@ -19,7 +19,10 @@
 
 def remap(model, strings):
     if isinstance(model, StringIndex):
-        return strings[model][1:]
+        if strings[model].startswith('*'):
+            # Strip the asterisk from a raw string.
+            return strings[model][1:]
+        return strings[model]
```

Comment:
Yeah, there's no decompress method in the code base simply because it isn't needed. It ought to be easy to implement here. I'm fairly sure the end result would be better.